<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MandelbulbUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
    <item>
     <layout class="QVBoxLayout" name="vL_LeftSideBar">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="buttonDeleteObject">
          <property name="toolTip">
           <string>Delete selected</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/cil-trash.svg</normaloff>:/icons/cil-trash.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="buttonShowObject">
          <property name="toolTip">
           <string>View selected</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/cil-camera.svg</normaloff>:/icons/cil-camera.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget_objectViewer">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <attribute name="headerCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>110</number>
        </attribute>
        <attribute name="headerHighlightSections">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_objectProperties">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Property</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="vL_Viewport" stretch="0">
      <item>
       <layout class="QHBoxLayout" name="hL_viewportToolbar">
        <item>
         <widget class="QToolButton" name="buttonToggleViewport">
          <property name="toolTip">
           <string>Toggle viewport</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/cil-screen-desktop_filled_custom.svg</normaloff>:/icons/cil-screen-desktop_filled_custom.svg</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuObject">
    <property name="title">
     <string>Object</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuGenerate">
    <property name="title">
     <string>Generate</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGenerate"/>
   <addaction name="menuObject"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExport_Entity_dat">
   <property name="text">
    <string>Export Entity (.dat)</string>
   </property>
  </action>
  <action name="actionImport_Entity_dat">
   <property name="text">
    <string>Import Entity (.dat)</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
